src/reactive_state_access.rs

```diff
@@ -1140,6 +1236,64 @@ mod test {
         0
     }
 
```

Comment:
The 3 unit tests also are failing , same way as for Reaction impl for `ObserveChangeReactiveState`